docs/modules/sort/overview.md

```diff
@@ -9,24 +9,34 @@ Inlong-sort is simply an Flink application, and relys on Inlong-manager to manag
 
 # features
 
+## supported transforms
+- String split index
```

Comment:
it's better to add an example to describe the transform operation.

Reply:
The demo of Transform is presented in dataflow_example.md